File: src/config/env.ts
import "dotenv/config.js";

const getEnv = (name: string): string => {
  const value = process.env[name];
  
  if (!value) {
    throw new Error(`Environment variable ${name} is not defined.`);
  }

  return value;
};

export const ENV = {
  DATABASE_URL: getEnv("DATABASE_URL"),
};

File: app/config/env.py
import os
from dotenv import load_dotenv

load_dotenv()


def get_env(name: str) -> str:
    value = os.getenv(name)

    if value is None:
        raise ValueError(f"Variável de ambiente {name} não encontrada")

    return value


DATABASE_URL = get_env("DATABASE_URL")